### ISSUE TITLE
Small typo

### ISSUE THREAD

**Issue by goldstar611** [concourse/concourse#7195]
I think there is a typo here:
https://github.com/concourse/concourse/blob/8e89cfce2b501b408b0cb841e6a2dab77b8ebe13/worker/workercmd/guardian.go#L59

The purpose of this `RemoveAll` call is to remove existing binaries that are older __than__ it is bundled with. Surely guardian doesn't bundle old binaries :P

**Comment by chenbh** [comment]
This is for when Concourse is being upgraded, the initial issue is at https://github.com/concourse/concourse/issues/6236. tldr: guardian doesn't overwrite old files when it extracts its bundled binaries, so if an older version of Concourse (+ guardian) was deployed on the system, the newer Concourse (+ guardian) would end up using the older binaries.

**Comment by goldstar611** [comment]
Yes I know, that's how I found this comment. But to be grammatical correct please s/that/than

**Comment by goldstar611** [comment]
To be clear I'm talking about the comment on line 59 not any code or logic. A tech writer would spot this in a few seconds.
